harness.sh

```diff
@@ -74,7 +74,7 @@ CLIENT_ADDRS=(
 
 # Number of mining clients to create. Maximum is determined by number of client
 # addresses above - currently 20.
-NUMBER_OF_CLIENTS=2
+NUMBER_OF_CLIENTS=1
```

Comment:
Deliberate change?

Reply:
Yeah the harness only needs a miner to work so it should probably start from there till more are needed for specific test cases.